### ISSUE TITLE
deterministic hashes example misses adding main index.[chunkhash].js to index.html

### ISSUE THREAD

**Issue by sean-stanley** [dangodev/webpack-optimize-sample-project#1]
I was following your awesome article and tips but got a bit stuck as my built code that attempted to add a manifest for long term caching was breaking my build. So I thought I'd try your repo and see if the issue exists there since our configuration was similar.

And indeed the issue exists here too. /manifest.json is imported as a script tag (which will generate an error) and index.[chunkhash].js is not added to index.html

Maybe this is an issue with a newer version of webpack or a newer version of html-webpack-plugin but I was curious if you had noticed this and found any fixes. 

I'm gonna keep looking and post my solution here once I find one.

**Comment by drwpow** [comment]
@sean-stanley you’re correct in that changes were made to webpack that have made the article—and these examples by extension—obsolete. The webpack docs on the subject have also changed dramatically over the past several months as well.

To compound the issue, many of the third-part webpack manifest plugins are now out of date, so it’s not a simple solution. But like you, I’ve been thinking about this issue specifically and I’d like to come up with a good solution. Let me tinker over the next few days and see if I can find the current best path forward.

It’s curious that such a critical part of optimization that is so hard to pull off in webpack, and not documented well.
